src/data_classes/TransformParser.gd

```diff
@@ -0,0 +1,21 @@
+class_name TransformParser extends RefCounted
+
+static func transform_to_text(transform: Transform2D) -> String:
+	var output := "matrix(%s, %s, %s, %s, %s, %s)"%[transform[0].x, transform[0].y, transform[1].x, transform[1].y, transform[2].x, transform[2].y]
+	return output
+
+static func text_to_transform(text: String) -> Transform2D:
+	if text.is_empty():
+		return Transform2D.IDENTITY
+	var formatted_text := text.strip_edges()
+	formatted_text = formatted_text.trim_prefix("matrix(")
+	formatted_text = formatted_text.trim_suffix(")")
```

Comment:
An initial implementation of this function must account for all possible syntaxes of transform, including ones like "translate(2 0) rotate(45 8 8)"

Reply:
https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform